unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Common/ShowHideUIByTriggerMinimap.cs

```diff
@@ -0,0 +1,14 @@
+using DCL;
+using UnityEngine;
+
+[RequireComponent(typeof(CanvasGroup))]
+public class ShowHideUIByTriggerMinimap : ShowHideUIByTrigger
+{
+
+    protected override void SetUIVisibility(bool isVisible)
+    {
+        if (!DataStore.i.HUDs.minimapVisible.Get())
+            return;
```

Comment:
Maybe ensure its visibility?
```suggestion
            { base.SetUIVisibility(false);
            return; }
```